cloud/pkg/controller/controller/downstream.go

```diff
@@ -189,8 +193,31 @@ func (dc *DownstreamController) syncEdgeNodes(stop chan struct{}) {
 			switch e.Type {
 			case watch.Added:
 				dc.lc.UpdateEdgeNode(node.ObjectMeta.Name)
+				fallthrough
 			case watch.Modified:
-				continue
+				// TODO: keep node cache for edge node to check if status has changed to Running from not running
+				// When node comes to running, send all the service information to edge
+				for _, nsc := range node.Status.Conditions {
+					if nsc.Type == "Ready" {
+						if nsc.Status == "True" {
+							msg := model.NewMessage("")
+							// TODO: what should in place of namespace and service when we are sending service list ?
+							resource, err := messagelayer.BuildResource(node.Name, "namespace", common.ResourceTypeServiceList, "service")
```

Comment:
we will send the entire list of services to the edgenode when the nodes comes to ready state from notReady. What should be the namespace and servicename field here ? as there will be  multiple services from different namespaces here